models/optional_modules/app_errors/app_errors.yml

```diff
@@ -99,8 +99,16 @@ models:
         description: '{{ doc("col_open_idfa") }}'
       - name: screen_id
         description: '{{ doc("col_screen_id") }}'
+        tests:
+          - not_null:
+              config:
+                enabled: '{{var("snowplow__enable_app_errors_module", false)}}'  
```

Comment:
I think both `screen_id` and `screen_name` come from the screen context which is optional and therefore these tests could fail. We could add an extra condition that `snowplow__enable_screen_context is true`